sdcm/utils/decorators.py

```diff
@@ -141,3 +144,46 @@ def wrapped(*args, **kwargs):
         return end - start, func_res
 
     return wrapped
+
+
+def latency_calculator_decorator(func):
+    """
+    Gets the start time, end time and then calculates the latency based on function 'calculate_latency'.
+
+    :param func: Remote method to run.
+    :return: Wrapped method.
+    """
+    from sdcm.utils import latency
```

Comment:
Why the import is it inside the function ?

Reply:
circular import

Reply:
Document it, otherwise some might try moving it